python/dgl/distributed/dist_dataloader.py

```diff
@@ -112,16 +66,12 @@ def __init__(self, dataset, batch_size, shuffle=False, collate_fn=None, drop_las
         self.pool, self.num_workers = get_sampler_pool()
         if queue_size is None:
             queue_size = self.num_workers * 4 if self.num_workers > 0 else 4
-        self.queue_size = queue_size
+        self.queue_size = queue_size  # prefetch size
         self.batch_size = batch_size
         self.num_pending = 0
         self.collate_fn = collate_fn
         self.current_pos = 0
-        if self.pool is not None:
-            m = mp.Manager()
-            self.queue = m.Queue(maxsize=queue_size)
-        else:
-            self.queue = Queue(maxsize=queue_size)
+        self.queue = []
```

Comment:
this is used when multiprocessing is disabled? if so, can you comment it?

Reply:
Added